### ISSUE TITLE
ingestSip - Success/Failure report?

### ISSUE THREAD

**Issue by kieranjol** [BAM-PFA/pymm#17]
it would be good to know if the script's run is a success or fail. The final report that I received was:
```
BUH-BYE
{'status': True, 'abortReason': '', 'ingestUUID': '496d9609-9830-4241-9d22-830eb8e48ad3', 'accessPath': '/Volumes/hi2u/kieran_tests/to_be_repacked/AF5841 Irish Holiday IFA-2013-34_lrp.mp4'}
```

So I'm not sure if all is well or not.

**Comment by mcampos-quinn** [comment]
Agreed, it's not really clear. The 'status' value in that result dict is the only marker for 'everything went ok' that shows up on the command line. I set it up initially to hook into a webapp that will be using this package, and that is looking for status=True, but maybe a simple, clearer printed message would be helpful? I'll look at it today.

**Comment by mcampos-quinn** [comment]
(lamely) addressed in 40d3c9e
It would be better to really think through how to present success/fail to the operator... eventually?

**Comment by kieranjol** [comment]
Haha I think it's perfect.
